pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbackrest_types.go

```diff
@@ -173,6 +173,12 @@ type PGBackRestRepoHost struct {
 	// +optional
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
 
+	// Topology spread constraints of a Dedicated repo host pod. Changing this
```

Comment:
👍🏻 This matches the description on Affinity. 🌱 Resources and Tolerations stick out now. We should pick one phrasing for "[dedicated] repo[sitory] host [pod]" and use it consistently.